package/README.md

```diff
@@ -47,3 +47,24 @@ Now run the build script:
 By default, the script will discover the Cargo workspace by walking up the directory hierarchy starting from the current working directory. You can manually specify the workspace root with the `--root-dir` argument.
 
 The script will create an `out` folder in the root of the Git repository containing the build artifacts.
+
+## Building Amazon Linux 2023 SRPM
+
+For Amazon Linux 2023 specifically, you can build an SRPM (Source RPM) package using the dedicated build script:
+
+    ./generate_amzn2023_srpm.sh
```

Comment:
This returns:
```
error: No such file or directory (os error 2)
```

The script seems to require to be invoked from the root of the repo.